core/src/test/java/io/questdb/test/griffin/SqlUtilTest.java

```diff
@@ -115,6 +117,106 @@ public void testImplicitCastStrAsLong256() {
         Assert.assertTrue(completed.await(TimeUnit.SECONDS.toNanos(2L)));
     }
 
+    @Test
+    public void testImplicitCastStrAsUuid() {
+        Uuid uuid = new Uuid();
```

Comment:
consider this:
```java
Uuid uuid = new Uuid();
SqlUtil.implicitCastStrAsUuid("00000000-0000-0002-0000-000000000001", uuid);
Assert.assertEquals(1, uuid.getLo());
Assert.assertEquals(2, uuid.getHi());
```
it's testing just the `implicitCastStrAsUuid()` as intended. it does not depend on `appendUuid()` being correct, etc. Also it makes the test more readable and obvious what is actually being tested.